Documentation/Readme_edihistory.html

```diff
@@ -21,8 +21,7 @@
 
 -->
 
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 
 <head>
```

Comment:
@tywrenn 

Such as this file, Is `<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">` needs to remove?

Reply:
I would double check the file (as in run it through W3C validator) and make sure that there aren't any XHTML tags or elements laying around as those would break the code in some cases. (I found this out the hard way lol)

Reply:
Ok, so should I modify commit or close the PR?

Reply:
@stu01509 I'd read up on how XHTML differs from HTML5. Then I would go through these files and make sure they don't contain anything XHTML related that could break code and if they do, i'd replace the offending code.